### ISSUE TITLE
Can not download or open any file from HTML download Attribute in android webview 

### ISSUE THREAD

**Issue by ankur-sardar** [react-native-community/react-native-webview#80]
If we go to this link - https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_a_download
And Click on the image (which is a download attribute HTML tag) , It does not download anything.

**Comment by ankur-sardar** [comment]
Should we support this feature in android webview? Create some Download listener in native side.
We can use `android.webkit.DownloadListener`

**Comment by jamonholmgren** [comment]
@ankur-sardar Could you edit your original issue body and provide an example of what you're trying to do that isn't working?

**Comment by vitorreis** [comment]
Hello there, I am not using this package yet (waiting for https://github.com/react-native-community/react-native-webview/pull/60 be merged), however I have the problem with the default webview in react-native, that nothing happens when the user clicks in any download link in the webview, nothing happens...

It seems that the same behaviour happens with this project as reported by @ankur-sardar ...
Any plans on implementing this feature?

**Comment by Titozzz** [comment]
This is a duplicate of #33 , will be fixed soon hopefully !

**Comment by ankur-sardar** [comment]
@Titozzz This issue is regarding the "Downloading" option for android webview (Not Upload as you referred the issue)
Are you going to support Downloading events also in #33 ?

**Comment by Titozzz** [comment]
Oh my bad, sorry, reopening, however we should wait for #33 to land before starting this I think. It will probably be way easier to implement than upload (I hope 😮)

**Comment by smathson** [comment]
@Titozzz @ankur-sardar see PR #203 re: download support on Android.

**Comment by smathson** [comment]
This has been implemented and released in 2.15.0. @Titozzz or @jamonholmgren, this issue can be closed.
